src/js/events/MetronomeActions.js

```diff
@@ -176,20 +178,42 @@ const MetronomeActions = {
     });
   },
 
-  suspendSchedule: function (jobID, data) {
+  disableSchedule: function (jobID, data) {
     RequestUtil.json({
       url: `${Config.metronomeAPI}/v1/jobs/${jobID}/schedules/${data.id}`,
       method: 'PUT',
       data,
       success: function () {
         AppDispatcher.handleServerAction({
-          type: REQUEST_METRONOME_JOB_SUSPEND_SUCCESS,
+          type: REQUEST_METRONOME_JOB_DISABLE_SCHEDULE_SUCCESS,
           jobID
         });
       },
       error: function (xhr) {
         AppDispatcher.handleServerAction({
-          type: REQUEST_METRONOME_JOB_SUSPEND_ERROR,
+          type: REQUEST_METRONOME_JOB_DISABLE_SCHEDULE_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          jobID,
+          xhr
+        });
+      }
+    });
+  },
+
+  enableSchedule: function (jobID, data) {
+    RequestUtil.json({
```

Comment:
Let's not introduce an `enableSchedule` method which is similar to the `disableSchedule` basically a general purpose update schedule method. Please see https://github.com/dcos/dcos-ui/pull/639 for details.